src/verify.c

```diff
@@ -0,0 +1,131 @@
+#include <openssl/bio.h>
+#include <openssl/cms.h>
+#include <openssl/err.h>
+#include <openssl/pem.h>
+#include <openssl/safestack.h>
+#include <openssl/x509.h>
+#include <openssl/x509_vfy.h>
+
+#include "./verify.h"
+
+return_t VF_init() {
+  ERR_load_crypto_strings();
+  OpenSSL_add_all_algorithms();
+  // Since none of these functions return useful error messages, per the
+  // openssl wiki documentation, we're just going to return success status.
+  // If this changes in future, we have the option to start failing
+  return VF_SUCCESS;
+}
+
+// TODO:
+//  - make sure all cleanup happens all the time
+//  - make sure that all return values are correct read
+//  - consider using ERR_get_error (iirc) to get actual error reasons
+//  - make sure we're correctly forcing FORMAT_PEM
```

Comment:
Are these done?  If not, best to file issues for them rather than leave inline TODO comments..

Reply:
Yes, I'll delete the list